Userland/Libraries/LibJS/Bytecode/Pass/DumpCFG.cpp

```diff
@@ -13,12 +13,12 @@ void DumpCFG::perform(PassPipelineExecutable& executable)
     started();
 
     VERIFY(executable.cfg.has_value());
-    outln(m_file, "CFG Dump for {} basic blocks:", executable.executable.basic_blocks.size());
```

Comment:
you can pass arguments to `PassManager::add<T>(...)`, which would be given to the ctor
Even if it didn't, instead of removing this, you could just add a default value for the file in the constructor

Reply:
Ah Ok, then it turns out that I was just to stupid to use it....